packages/bundle-source/README.md

```diff
@@ -42,6 +42,14 @@ entry instead of `main` in `package.json`, if not overridden by explicit
 The `development` condition additionally implies that the bundle may import
 `devDependencies` from the package containing the entry module.
 
+## Comment Elision
+
+The `--no-comment` (`-X`) flag with `--format` `endoScript` or `endoZipBase64`
```

Comment:
this is a cute name but "no" sounds like opting _out_  of behavior, not into one.

The name should convey that it's opting in. E.g. `strip-comments` as is the common name for this behavior or `elide-comments` as is the term used throughout the implementation here.

Reply:
Totally worth the gag, even if lost in a fit of responsibility. I want to reserve `--strip-comments` for a feature that removes the comment brackets entirely. Please push back if that is undue pedantry and I’ll thread `stripComments` throughout.